tests/test_admin.py

```diff
@@ -847,6 +847,27 @@ class ArchiveViewTestCase(BaseStateTestCase):
     def setUp(self):
         self.versionable = PollsCMSConfig.versioning[0]
 
+    @patch('django.utils.timezone.now')
+    @patch('django.contrib.admin.ModelAdmin.message_user')
+    def test_archive_view_sets_modified_time(self, mocked_messages, mocked_time):
+        mocked_time.return_value = datetime.datetime(2099, 1, 1, 11, 1, 11, 1111, tzinfo=utc)
+        poll_version = factories.PollVersionFactory(state=constants.DRAFT)
+
+        url = self.get_admin_url(
+            self.versionable.version_model_proxy, 'archive', poll_version.pk)
+
+        user = self.get_staff_user_with_no_permissions()
+
+        with self.login_user_context(user):
+            self.client.post(url)
+
+        # get object again after state change
+        poll_version_ = Version.objects.get(pk=poll_version.pk)
+        # check time is not same as before
+        self.assertNotEqual(poll_version_.modified, poll_version.modified)
```

Comment:
What are you achieving in this assert that you're not achieving with the assert below? To me it looks suspiciously like you're testing if you're mocking worked rather than whether the field is being updated correctly. If you do the mocking of this with freezegun you can be confident it's being done ok :)